server/mdm/apple/apple_mdm.go

```diff
@@ -526,11 +526,9 @@ func (d *DEPService) processDeviceResponse(ctx context.Context, depClient *godep
 		logs = append(logs, logCountsForResults(apiResp.Devices)...)
 		level.Info(logger).Log(logs...)
 
-		debugLogs := []interface{}{"msg", "assign profile responses by device"}
-		for k, v := range apiResp.Devices {
-			debugLogs = append(debugLogs, k, v)
+		if err := d.ds.UpdateHostDEPAssignProfileResponses(ctx, apiResp); err != nil {
+			return ctxerr.Wrap(ctx, err, "update host dep assign profile responses")
 		}
-		level.Debug(logger).Log(debugLogs...)
```

Comment:
curious: any reason to not keep the logs?

Reply:
To keep it a little cleaner here, I moved the logging into the db [layer](https://github.com/fleetdm/fleet/pull/16973/files#diff-2f8a6d69f21403d6845da98a6a9a7e3808f275a21ec0209cbabcdbeaae57676eR2919. There's a log line with the number of devices for each profile_uuid+status combo. We could log the serials instead of the counts, but I wan't sure how visible we wanted to have serials in the logs.